src/test/java/org/fundacionjala/coding/norman/banck_ocr/BankOcrTest.java

```diff
@@ -0,0 +1,239 @@
+package org.fundacionjala.coding.norman.banck_ocr;
+
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by NORMAN on 2/7/2017.
+ */
+public class BankOcrTest {
+    /**
+     * Test if testBankOcrWhenScannedNumbersCero.
+     */
+    @Test
+    public void testBankOcrWhenScannedNumbersCero() {
+        // given:
+        List<String> insertLineFour = Arrays.asList(
+                " _ "
+                        + "| |"
+                        + "|_|"
+        );
+
+        // when:
+        HistoryOneBankOcr bankOcr = new HistoryOneBankOcr();
+        String actualResult = bankOcr.verificationLineNumber(insertLineFour);
+
+        // then:
+        String expectedResult = "0";
+        assertEquals(expectedResult, actualResult);
+    }
+
+    /**
+     * Test if testBankOcrWhenScannedNumbersCeroToNine.
+     */
+    @Test
+    public void testBankOcrWhenScannedNumbersCeroToNine() {
+        // given:
+        List<String> insertLineFour = Arrays.asList(
+                " _ "
+                        + "| |"
+                        + "|_|",
+
+                "  |"
+                        + "  |"
+                        + "  |",
+
+                " _ "
+                        + " _|"
+                        + "|_ ",
+
+                "__ "
+                        + " _|"
+                        + "__|",
+
+                "   "
+                        + "|_|"
+                        + "  |",
+
+                " __"
+                        + "|__"
+                        + " __|",
+
+                " __"
+                        + "|__"
+                        + "|__|",
+
+                "__ "
+                        + "  |"
+                        + "  |",
+
+                " _ "
+                        + "|_|"
+                        + "|_|",
+
+                " _ "
+                        + "|_|"
+                        + " _|"
+        );
+
+        // when:
+        HistoryOneBankOcr bankOcr = new HistoryOneBankOcr();
```

Comment:
You are repeating this line in all your tests
Please move it in a @Before method